scripts/user-actions/display.js

```diff
@@ -3,14 +3,13 @@ console.log('--- loading: display-all.js');
 
 // reverse engineering!
 //   the handler has no tests, study the demo to write this function
-const displayHandler = (message = '', arrayToDisplay = words) => {
+ const displayHandler = (message = '', arrayToDisplay = words) => {
   debugger;
   const combineStrings = (acc, next) => {
-    return _;
+    return acc+'\n'+next;
   }
-  const alertMessage = arrayToDisplay
-    ._(_, _);
```

Comment:
You need to use a method, so all of our messages will display. What do you think @LujiAnna @julia-sod @galalkoro

Reply:
Yes @harunaltunhr , you're right, here should be used a reduce method

Reply:
thank you for your reply , now i do like the example